mantidproject/mantid
# Replaced autoscale button with a checkbox in muon gui

**Description of work.**
Autoscale button in plotting window was replaced with a checkbox


**To test:**
1. Open Muon analysis gui.
2. Load suitable data.
3. Check autoscale checkbox on plotting window and plot limits should 
be modified accordingly and y selection range should be disabled.
4. Changing dataset while autoscale checkbox is still checked set limits of 
plot accordingly.
5. Clicking on the pan or zoom button should uncheck the autoscale checkbox.
6. The same should hold for frequency domain analysis gui.

Fixes #29065.


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
